engine/exchange_test.go

```diff
@@ -42,6 +39,51 @@ func CleanupTest(t *testing.T) {
 	}
 }
 
+func TestExchangeManagerAdd(t *testing.T) {
+	t.Parallel()
+	var e exchangeManager
+	bitfinex := new(bitfinex.Bitfinex)
+	bitfinex.SetDefaults()
+	e.add(bitfinex)
+	if exch := e.getExchanges(); exch[0].GetName() != "Bitfinex" {
+		t.Error("unexpected exchange name")
+	}
+}
+
+func TestExchangeManagerGetExchanges(t *testing.T) {
+	t.Parallel()
+	var e exchangeManager
+	if exchanges := e.getExchanges(); exchanges != nil {
+		t.Error("unexpected value")
+	}
+	bitfinex := new(bitfinex.Bitfinex)
+	bitfinex.SetDefaults()
+	e.add(bitfinex)
+	if exch := e.getExchanges(); exch[0].GetName() != "Bitfinex" {
+		t.Error("unexpected exchange name")
+	}
+}
+
+func TestExchangeManagerRemoveExchange(t *testing.T) {
+	t.Parallel()
+	var e exchangeManager
+	if err := e.removeExchange("Bitfinex"); err != ErrNoExchangesLoaded {
+		t.Error("no exchanges should be loaded")
+	}
+	bitfinex := new(bitfinex.Bitfinex)
+	bitfinex.SetDefaults()
+	e.add(bitfinex)
+	if err := e.removeExchange(testExchange); err != ErrExchangeNotFound {
+		t.Error("Bitstamp exchange should return an error")
```

Comment:
Bitfinex?

Reply:
Nope, we loaded Bitfinex and Bitstamp shouldn't be loaded